db/compaction/compaction_job.cc

```diff
@@ -450,7 +412,11 @@ void CompactionJob::Prepare() {
     RecordInHistogram(stats_, NUM_SUBCOMPACTIONS_SCHEDULED,
                       compact_->sub_compact_states.size());
   } else {
-    compact_->sub_compact_states.emplace_back(c, nullptr, nullptr);
+    constexpr Slice* start = nullptr;
+    constexpr Slice* end = nullptr;
+    constexpr uint64_t size = 0;
```

Comment:
Is it for providing name of parameters of line 419?

Reply:
Yes, it's kind of like a C++11 equivalent of putting the parameter names there as `/* ... */` comments.